integration-test/src/test/java/com/datastrato/gravitino/integration/test/trino/TrinoQueryIT.java

```diff
@@ -185,19 +235,23 @@ private static void createCatalog(
     }
 
     boolean catalogCreated = false;
-    int tries = 30;
+    int tries = 180;
     while (!catalogCreated && tries-- >= 0) {
-      String result = trinoQueryRunner.runQuery("show catalogs");
-      if (result.contains(metalakeName + "." + catalogName)) {
-        catalogCreated = true;
-        break;
+      try {
+        String result = trinoQueryRunner.runQuery("show catalogs");
+        if (result.contains(metalakeName + "." + catalogName)) {
+          catalogCreated = true;
+        }
+        LOG.info("Waiting for catalog {} to be created", catalogName);
```

Comment:
At this point, the catalog has been created, so it's not the 'Waiting for ...', right?